src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -229,17 +227,7 @@ private void DisposeVisualStudioDocumentTrackingService()
 
         private void LoadAnalyzerNodeComponents()
         {
-            _analyzerTracker = this.ComponentModel.GetService<AnalyzerItemsTracker>();
-            if (_analyzerTracker != null)
-            {
-                _analyzerTracker.Register();
-            }
-
-            var analyzerCommandHandler = this.ComponentModel.GetService<AnalyzersCommandHandler>();
-            if (analyzerCommandHandler != null)
-            {
-                analyzerCommandHandler.Initialize((IMenuCommandService)this.GetService(typeof(IMenuCommandService)));
-            }
+            this.ComponentModel.GetService<IAnalyzerNodeSetup>().Initialize(this);
```

Comment:
Previously, this code was in a project that referenced both the SolutionExplorerShim and ServicesVisualStudio That's why it was able to see the types directly. Now that it's in ServicesVisualStudio, it can't see the types in SolutionExplorerShim since there's already a project reference moving the other way.

I discussed this with @srivatsn and we agreed it might be possible to move these required types into ServicesVisualStudio, but I didn't want to risk breaking the subtleties here around perf and break something.